windows/scripts/MatlabPostBuild.cmd

```diff
@@ -0,0 +1,12 @@
+set SOLUTION_DIR=%~1%
+set OUTPUT_DIR=%~2%
+
+echo MatlabPostBuild.cmd : copy matlab generated scripts to output.
+
+copy /y "%SOLUTION_DIR%..\matlab\+caffe\*.m" "%OUTPUT_DIR%matcaffe\+caffe"
```

Comment:
Is it necessary to copy all of this to build folder? I think that `classification_demo` and `hdf5creation` are not needed, as they are just examples. `+caffe\imagenet` seems to be only used by classification demo, so I would drop that as well.

I'm not sure about copying unit tests. We don't do it in Python. Is there a way to execute unit tests easily if only `+caffe\private` is copied to Release?

Also, I notice that `+caffe\private\caffe_.cpp` is also copied; that should definitely be excluded :)